pages/dkp/kommander/2.1/release-notes/2.1.2/index.md

```diff
@@ -0,0 +1,277 @@
+---
+layout: layout.pug
+navigationTitle: Kommander 2.1.2 Release Notes
+title: Kommander 2.1.2 Release Notes
+menuWeight: 30
+excerpt: View release-specific information for Kommander 2.1.2
+enterprise: false
+beta: false
+---
+**D2iQ&reg; Kommander&reg; version 2.1.2 was released on May 11, 2022.**
+
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Kommander[/button]
+
+To get started with Kommander, [download](../../download/) and [install](../../install/) the latest version of Kommander.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+## Release summary
+
+This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in Kommander.
+
+## Fixes and Improvements
```

Comment:
Do we like these fixes as headings or just a bulleted list? I don't care, other than consistency.

Reply:
Example of the other way - https://docs.d2iq.com/dkp/konvoy/1.8/release-notes/1.8.1/

Reply:
The list was long enough, I thought this might scan quicker, but I'm not set on either way.
Anyone feel strongly about this?

Reply:
Since we are adding a bit more context to each of the fixes, I think we should keep them as headings?